mlflow/projects/__init__.py

```diff
@@ -27,7 +27,7 @@
 _GIT_URI_REGEX = re.compile(r"^[^/]*:")
 # Environment variable indicating a path to a conda installation. MLflow will default to running
 # "conda" if unset
-MLFLOW_CONDA = "MLFLOW_MLFLOW_CONDA"
+MLFLOW_CONDA = "MLFLOW_CONDA"
```

Comment:
Is this intended? This seems like a backwards-incompatible change, if it's in effect, and probably deserves a PR of its own (however small).

Reply:
Yeah it's intended, looks like I made a typo in an earlier PR - we haven't released an MLflow version with this change yet so thankfully the typo hasn't messed anything up. That being said, good call on putting it in a separate PR - I'll do that